### PR TITLE
OSDOCS-4752 Pre-GA docs nit corrections--MicroShift

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
[OSDOCS-4752](https://issues.redhat.com//browse/OSDOCS-4752) Pre-GA docs nit corrections--MicroShift

This PR applies some copy edits and corrections made by the MicroShift engineering and docs teams. It does not substantively change any previously approved content.

Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OSDOCS-4752

Docs preview: 
https://54323--docspreview.netlify.app/microshift/latest/microshift_getting_started/microshift-understanding.html#con-about-microshift_microshift-understanding
https://54323--docspreview.netlify.app/microshift/latest/microshift_getting_started/microshift-architecture.html
https://54323--docspreview.netlify.app/microshift/latest/microshift_install/microshift-install-rpm.html
https://54323--docspreview.netlify.app/microshift/latest/microshift_install/microshift-install-rpm.html#preparing-install-microshift-from-rpm-package_microshift-install-rpm
https://54323--docspreview.netlify.app/microshift/latest/microshift_install/microshift-install-rpm.html#accessing-microshift-cluster-locally_microshift-install-rpm
https://54323--docspreview.netlify.app/microshift/latest/microshift_install/microshift-install-rpm.html#system-requirements-installing-microshift
https://54323--docspreview.netlify.app/microshift/latest/microshift_troubleshooting/microshift-version.html#microshift-version-cli_microshift-version

QE review:
N/A -- MicroShift is Developer Preview
SME approval is required (given below)

Additional information:
Not many content changes, but a lot of small corrections (like deleting extra spaces) and a renaming of several modules so that they all begin with "microshift-" and are easier to find in the repo.

### CODE REVIEW

**Comment by ShaunaDiaz** [comment]
@dhellmann, @ggiguash, PTAL and let me know if looks good to you.
@robin-owen PTAL and drop in a review comment for where and what to add to fix those version #s in the commands that were discussed in the Google doc (and anything else you come across).

File: modules/accessing-microshift-cluster-remotely.adoc [hunk 1]
```diff
@@ -1,48 +1,48 @@
...
 ====
-The following procedure assumes that there are two users. The first user, `user@microshift`, is the user on the {product-title} host machine. `user@microshift` is responsible for setting up the `op-system` machine so that it can be accessed by a remote user on a separate workstation.  
+The following procedure assumes that there are two users. The first user, `user@microshift`, is the user on the {product-title} host machine. `user@microshift` is responsible for setting up the `op-system` machine so that it can be accessed by a remote user on a separate workstation.
 
-The second user, `user@workstation`, accesses the {op-system} machine remotely from their workstation. 
+The second user, `user@workstation`, accesses the {op-system} machine remotely from their workstation.
 ====
 
-. As `user@microshift` on the MicroShift host, open the firewall port for Kubernetes API server (`6443/tcp`) on your {op-system} machine by entering the following command:
+. As `user@microshift` on the {product-title} host, open the firewall port for Kubernetes API server (`6443/tcp`) on your {op-system} machine by entering the following command:
```
Comment (dhellmann) [inline modules/accessing-microshift-cluster-remotely.adoc:24]:
```suggestion
. As `user@microshift` on the {product-title} host, open the firewall port for Kubernetes API server (`6443/tcp`) by entering the following command:
```

nit: We could remove the redundant phrase indicating which host we expect the user to be on.

File: modules/preparing-for-installing-microshift-rpm.adoc [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
+{product-title} uses the logical volume manager storage (LVMS) Container Storage Interface (CSI) provider for storing PVs. LVMS relies on Linux's LVM to dynamically manage the backing storage for PVs. For this reason, your machine must have an LVM VG with unused space in which LVMS can create the LVM logical volumes (LVs) for your workload's PVs.
 
-To configure an LVM VG that allows LVMS to create the LVM LVs for your workload's PVs, adjust your root volume's size during the installation of {op-system}. Adjusting your root volume's size provides free space for additional LVs created by LVMS at runtime. 
+To configure an LVM VG that allows LVMS to create the LVM LVs for your workload's PVs, adjust your root volume's size during the installation of {op-system}. Adjusting your root volume's size provides free space for additional LVs created by LVMS at runtime.
```
Comment (dhellmann) [inline modules/preparing-for-installing-microshift-rpm.adoc:12]:
```suggestion
To configure a volume group that allows LVMS to create the logical volumes for your workload's PVs, adjust your root volume's size during the installation of {op-system}. Adjusting your root volume's size provides free space for additional LVs created by LVMS at runtime.
```

nit: There's  nothing technically wrong with the current text, but it's not necessary to specify that a volume group or physical volume is related to LVM (they are by definition). Spelling the words out feels clearer.

**Review by dhellmann — COMMENTED** [review]
These changes all look good. I had 2 nits, but they're just suggestions so feel free to ignore them.

/lgtm

File: modules/preparing-for-installing-microshift-rpm.adoc (hunk 2 above)
Comment (ShaunaDiaz) [reply modules/preparing-for-installing-microshift-rpm.adoc:12]: Agreed, spelled out volume group and defined (VG), but we defined LVs just above there, so I made that one just LVs.

**Comment by kalexand-rh** [comment]
@ShaunaDiaz, please post links to the topics that have extensive changes within the netlify preview. While the changes are scattered, some of them look extensive, and it's a big help for the reviewers to know where to start.

**Comment by ShaunaDiaz** [comment]
@kalexand-rh Links updated.

**Review by mjpytlak — COMMENTED** [review]
Hi @ShaunaDiaz just a handful of comments for your consideration. Thank you.

**Comment by ShaunaDiaz** [comment]
6bc325d Makes a folder-name correction noted on a review of another piece that uses the same module.

**Comment by mburke5678** [comment]
@ShaunaDiaz ^^

File: modules/microshift-installing-from-rpm.adoc [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+= Installing {product-title} from an RPM package
 
-Use the following procedure to install MicroShift from an RPM package. 
+Use the following procedure to install {product-title} from an RPM package.
 
-.Prerequisites 
+.Prerequisites
 
-* The xref:../microshift_install/microshift-install-rpm.adoc#system-requirements-installing-microshift[system requirements for installing MicroShift] have been met. 
+* The xref:../microshift_install/microshift-install-rpm.adoc#system-requirements-installing-microshift[system requirements for installing MicroShift] have been met.
```
Comment (mburke5678) [inline modules/microshift-installing-from-rpm.adoc:13]: No links in modules

Comment (mjpytlak) [reply modules/microshift-installing-from-rpm.adoc:13]: @mburke5678 thanks for catching these. I completely overlooked them.

**Comment by ShaunaDiaz** [comment]
@mburke5678 I'm fixing them now.

**Pull request merged**